src/pages/welcome/index.js

```diff
@@ -25,7 +26,11 @@ const Welcome = () => {
 	};
 
 	const onComplete = () => {
-		onCreateProfile(profile.firstName, profile.lastName, profile.githubUsername, profile.bio);
+		if (profile.profilePicture) {
+			onCreateProfile(profile.firstName, profile.lastName, profile.githubUsername, profile.bio, profile.profilePicture);
```

Comment:
you could update `onCreateProfile` to check if any fields are undefined not to include them in the api call, so here you can just pass profilePicture even if empty/missing and no need to duplicate the if... else...